test/src/main/java/org/apache/accumulo/test/compaction/CompactionExecutorIT.java

```diff
@@ -67,36 +69,49 @@
 import org.apache.accumulo.core.spi.compaction.CompactionKind;
 import org.apache.accumulo.core.spi.compaction.CompactionPlan;
 import org.apache.accumulo.core.spi.compaction.CompactionPlanner;
+import org.apache.accumulo.harness.MiniClusterConfigurationCallback;
 import org.apache.accumulo.harness.SharedMiniClusterBase;
+import org.apache.accumulo.minicluster.ServerType;
+import org.apache.accumulo.miniclusterImpl.MiniAccumuloConfigImpl;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.RawLocalFileSystem;
 import org.apache.hadoop.io.Text;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-@Disabled // ELASTICITY_TODO
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+
 public class CompactionExecutorIT extends SharedMiniClusterBase {
+  public static final List<String> compactionGroups = new LinkedList<>();
 
   public static class TestPlanner implements CompactionPlanner {
 
+    private static class ExecutorConfig {
+      String name;
+      String type;
```

Comment:
Does not seem like these two fields are read, but not sure.  If they are not read could they be dropped?

Reply:
Yes they can be dropped.